scripts/hail_batch/hgdp1kg_tobwgs_densified_pca_new_variants/hgdp_1kg_tob_wgs_densified_pca_new_variants.py

```diff
@@ -0,0 +1,63 @@
+"""
+Perform PCA on densified TOB-WGS data. Reliant on output from
+```
+hgdp1kg_tobwgs_densified_pca_new_variants/
+hgdp_1kg_tob_wgs_densified_pca_new_variants.py
+````
+"""
+
+import hail as hl
+import pandas as pd
+from hail.experimental import lgt_to_gt
+from analysis_runner import bucket_path, output_path
+
+
+TOB_WGS = bucket_path('1kg_hgdp_densify_new_variants/v0/tob_wgs_filtered.mt/')
+GNOMAD_HGDP_1KG_MT = (
+    'gs://gcp-public-data--gnomad/release/3.1/mt/genomes/'
+    'gnomad.genomes.v3.1.hgdp_1kg_subset_dense.mt'
+)
+
+
+def query():
+    """Query script entry point."""
+
+    hl.init(default_reference='GRCh38')
+
+    tob_wgs = hl.read_matrix_table(TOB_WGS)
+    hgdp_1kg = hl.read_matrix_table(GNOMAD_HGDP_1KG_MT)
+
+    # keep loci that are contained in the densified, filtered tob-wgs mt
+    hgdp_1kg = hgdp_1kg.semi_join_rows(tob_wgs.rows())
+
```

Comment:
This is already implied in the `union_cols` call below.

```suggestion
```

Reply:
That's true, but I wanted to keep that since I use `hgdp_1kg.cols()` as metadata information (and if I do a `union_cols` only, that removes the column information). I use this for plotting downstream (e.g., continental populations, etc.)

Reply:
👍